lib/task/arCacheDescriptionXmlTask.class.php

```diff
@@ -32,7 +32,8 @@ protected function configure()
       new sfCommandOption('application', null, sfCommandOption::PARAMETER_OPTIONAL, 'The application name', 'qubit'),
       new sfCommandOption('env', null, sfCommandOption::PARAMETER_REQUIRED, 'The environment', 'cli'),
       new sfCommandOption('connection', null, sfCommandOption::PARAMETER_REQUIRED, 'The connection name', 'propel'),
-      new sfCommandOption('skip', null, sfCommandOption::PARAMETER_OPTIONAL, 'Number of information objects to skip', 0)
+      new sfCommandOption('skip', null, sfCommandOption::PARAMETER_OPTIONAL, 'Number of information objects to skip', 0),
+      new sfCommandOption('limit', null, sfCommandOption::PARAMETER_OPTIONAL, 'Number of information objects to export', 0)
```

Comment:
Does `limit = 0` mean no limit using `Propel::getDB()->applyLimit`? Otherwise, I'd change the default.

Reply:
It does, yeah. Maybe I'll try changing the default for it to null as that would be a bit clearer.

Reply:
Maybe it's better to give the option of no limit like it's then.

Reply:
Seems like it works the same with null as a default.